examples/shared_samples/logical_interconnect_group.rb

```diff
@@ -161,10 +165,11 @@
 puts "#{type} was updated successfully:"
 puts lig.data
 
+puts eth01['uri']
 if variant == 'Synergy'
   puts "\nAdding an internal network with uri = #{eth01['uri']}"
   lig.retrieve!
-  lig.add_internal_network(eth01)
```

Comment:
Is there any reason to remove this line..? If it gives method unavailable error, then it is because that "add_internal_network" method is available only for synergy module. So to avail that method, you have to declare the lig_class with synergy variant. I have mentioned the usage in above comment.